lib/rubocop/config_loader.rb

```diff
@@ -184,7 +184,8 @@ def yaml_safe_load(yaml_code, filename)
         if defined?(SafeYAML) && SafeYAML.respond_to?(:load)
           SafeYAML.load(yaml_code, filename,
                         whitelisted_tags: %w[!ruby/regexp])
-        elsif RUBY_VERSION >= '2.6'
+        # Ruby 2.6+
+        elsif Gem::Version.new(Psych::VERSION) >= Gem::Version.new('3.1.0.pre1')
```

Comment:
Cool version comparison trick! I've never seen this before!

Reply:
I learned this comparison at https://github.com/rubygems/rubygems/pull/2439/files.